src/demo-app/progress-bar/progress-bar-demo.scss

```diff
@@ -0,0 +1,12 @@
+.demo-progress-bar-container {
+  width: 100%;
+}
+
+md-progress-bar {
```

Comment:
Can you do this with a class instead and also prefix all the classes with `demo-` ?